README.md

```diff
@@ -49,12 +52,12 @@ pod 'Sentinel'
 *Sentinel* is made of multiple subspecs:
 - `Core` which will install only the core features for the *Sentinel* to be usable
 - `UserDefaults` which will add the `UserDefaultsTool`
-- `EmailSendel` which will add the `EmailSenderTool`
+- `EmailSender` which will add the `EmailSenderTool`
 - `CustomLocation` which will add the `CustomLocationTool`
 - `TextEditing` which will add the `TextEditingTool`
-- `Default` - which will install `UserDefaults`, `Core`, `CustomLocation`, and `TextEditing`
+- `Default` - which will install `UserDefaults`, and `TextEditing`
```

Comment:
Based on this list, I would recommend adding a section like; "Available tools" and list all of them with info about platform support and one sentence description.

And for CocoaPods and SPM, just add info if necessary - e.g., for CocoaPods it can stay as is, while notes about macOS support can be removed because that would be mentioned already.